docs/modules/ROOT/pages/references/secrets.adoc

```diff
@@ -0,0 +1,175 @@
+= Secret References
+
+To ensure maximum compatibility between the various service providers, we ensure that the secrets are as consistent as possible within a given service type.
+Due to differences between service providers it might be possible that the access secrets aren't 100% interchangeable.
+
+== PostgreSQL
+
+The secret contains the plaintext values for the given keys.
+
+// tag::postgres[]
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: postgres-creds
+data:
+  POSTGRESQL_DB: postgres <1>
+  POSTGRESQL_HOST: my-postgres-example.my-cloud.com <2>
+  POSTGRESQL_PASSWORD: my-secret <3>
+  POSTGRESQL_PORT: 21699 <4>
+  POSTGRESQL_URL: postgres://postgres:my-secret@my-postgres-example.my-cloud.com:21699/postgresql?sslmode=require <5>
+  POSTGRESQL_USER: postgres <6>
+  ca.crt: | <7>
+    -----BEGIN CERTIFICATE-----
+    HexValues
+    -----END CERTIFICATE-----
+----
+<1> Database name
+<2> Host to connect to
+<3> Password
+<4> Port
+<5> URL containing all necessary information to connect to the instance
+<6> Username
+<7> ca.crt to use when using `sslmode=verify-full`
+
+IMPORTANT: All data fields are Base64-encoded, like any other https://kubernetes.io/docs/tasks/configmap-secret/managing-secret-using-kubectl/[Kubernetes secret].
```

Comment:
IMHO we could remove that, this should be "common knowledge" for Kubernetes users...